### PR TITLE
colorful console and go fmt

### PR DESCRIPTION
**Pull request by JessonChan**
go fmt  makes the code more **go**

### CODE REVIEW

File: logs/console.go [hunk 1]
```diff
@@ -68,7 +68,11 @@ func (c *consoleWriter) Init(jsonConfig string) error {
 	if len(jsonConfig) == 0 {
 		return nil
 	}
-	return json.Unmarshal([]byte(jsonConfig), c)
+	err := json.Unmarshal([]byte(jsonConfig), c)
+	if runtime.GOOS == "windows" {
```
Comment (JessonChan) [inline logs/console.go]: if the os is windows,the color filed will be useless

File: config.go [hunk 2]
```diff
@@ -120,12 +119,8 @@ var (
 
 func init() {
 	AppPath, _ = filepath.Abs(filepath.Dir(os.Args[0]))
-	workPath, _ = os.Getwd()
-	workPath, _ = filepath.Abs(workPath)
 
-	if workPath != AppPath {
-		os.Chdir(AppPath)
-	}
+	os.Chdir(AppPath)
```
Comment (astaxie) [inline config.go]: current workdir is different with apppath

Comment (JessonChan) [inline config.go]: but os.Chdir(AppPath) will change current work dir to AppPath

File: config.go [hunk 3]
```diff
@@ -120,12 +119,8 @@ var (
 
 func init() {
 	AppPath, _ = filepath.Abs(filepath.Dir(os.Args[0]))
-	workPath, _ = os.Getwd()
```
Comment (JessonChan) [inline config.go]: @youngsterxyf have a look here,thanks

Comment (youngsterxyf) [inline config.go]: I think the change at this position is OK. However, after starting, should beego app change current dir to AppPath? We should make a explicit hypothesis - relative path settings is relative to AppPath or current path - this decide that beego should change current dir to AppPath or not.

Comment (JessonChan) [inline config.go]: should change to AppPath for some configure files,and workPath is useless in fact.

**Pull request merged**